docs/contributing/translations.md

```diff
@@ -0,0 +1,36 @@
+# Translations
+Wagtail uses Transifex to translate the content for the admin interface. Our goal is to ensure that Wagtail can be used by those who speak many different languages. Translation of admin content is a great way to contribute without needing to know how to write code.
+
+## Translation workflow 
+Translations from Transifex are only integrated into the Wagtail repository at the time of a new release. When a release is close to being ready there will be a RC (Release Candidate) for the upcoming version and the translations will be exported to Transifex.
+
+During this RC period, usually around two weeks, there will be a chance for all the translators to update and add new translations. We will also notify the `#translators` channel in the Watail slack group at this time.
+
+These new translations are imported into Wagtail for any subsequent RC and the final release.
+
+If translations reach a threshold of about 80% we communicate that this is an officially supported translation in the docs.
+
+## What is Transifex
+ Wagtail is localised (translated) using [Transifex](https://www.transifex.com/), a web platform that helps organisations coordinate translation projects.
+
+## To help out in translating wagtail here's what you will do
+- Introduce yourself on [slack]( https://wagtailcms.slack.com/archives/CTKN6UXKN) to join fellow translators on slack.
+- Request access to become a member of the language team you want to work with on slack. Mention your Transifex username when you make this request
+-	Go to [Transifex](https://www.transifex.com/)
+-	Click on start for free
+-	Fill in your Username, Email and Password
+-	Agree to the terms and conditions
+-	Click on free trial or join an existing organisation
+-	Join [wagtail](https://www.transifex.com/torchbox/wagtail/dashboard/)
+-	A list of project languages is provided on the [wagtail](transifex.com\torchbox\wagtail\dashboard\) dashboard
+-	A view resources button appears when you hover over the ready to use part on the right side of the page
+-	Click on the button to get access to the resources available
+-	This takes you to the language section
+-	This page has a translation panel on the right and a list of strings to be translated on the left
+-	To translate a project, select it and enter your translation in the translation panel
+-	Save the translation using the translation button on the panel
+Here is a [link](https://youtu.be/sLI_AuOMUQw?t=17) to a video that walks you through using Transifex step-by-step
```

Comment:
```suggestion
Here is a [Wagtail Space US 2020 Lightning Talk](https://www.youtube.com/watch?v=sLI_AuOMUQw&t=17s) that walks through using Transifex step-by-step.
```

Links should not use the word 'link' or 'here' but rather be super clear where the link is going to, also best to use the closest canonical URL (not a shortened URL).